tests/Avalonia.ReactiveUI.UnitTests/Avalonia.ReactiveUI.UnitTests.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You need to add this project [here](https://github.com/AvaloniaUI/Avalonia/blob/master/build.cake#L172)